xbmc/dialogs/GUIDialogPlayEject.cpp

```diff
@@ -66,14 +81,16 @@ bool CGUIDialogPlayEject::OnMessage(CGUIMessage& message)
 
 void CGUIDialogPlayEject::FrameMove()
 {
-  CONTROL_ENABLE_ON_CONDITION(ID_BUTTON_PLAY, g_mediaManager.IsDiscInDrive());
+  if (isDisc)
+    stubOriginal = URIUtils::GetOpticalMediaPath();
+  CONTROL_ENABLE_ON_CONDITION(ID_BUTTON_PLAY, CFile::Exists(stubOriginal));
```

Comment:
honestly I don't know too much about kodi dialogs. This is how already works right now with disc stubs. Everytime framemove gets called kodi checks if the disc is in the drive. Is FrameMove called every frame?